windows/utils/win_disable_bitlocker.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022-2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Disable BitLocker service and decrypt the volumes
+# Parameters:
+#   decrypt_wait_time: the time in seconds to wait for the volume decryption.
+#
+- name: "Initialize the decryption wait time"
+  ansible.builtin.set_fact:
+    decrypt_wait_time: 900
+  when: decrypt_wait_time is undefined or not decrypt_wait_time
+
+- name: "Stop and disable BitLocker service in guest OS"
+  include_tasks: win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "Set-Service -Name BDESVC -Status stopped -StartupType disabled"
```

Comment:
From the settings of `need_disable_bitlocker`, as long as any one of `bitlocker_volume_list | length > 0` or `service_status == 'Running'` is true, then `need_disable_bitlocker` will be true. That means, one of them could be false but the other one is true.

If bitlocker service is not running, this task can be skipped as it's already disabled. 
If bitlocker volume list is empty, below decryption can be skipped as no volume needs decryption.

That's why I suggest you moving the checking task into this file.